prov/rxm/src/rxm.h

```diff
@@ -60,6 +60,7 @@
 #define RXM_MINOR_VERSION 0
 
 #define RXM_BUF_SIZE 16384
+#define RXM_BUF_CNT 1
```

Comment:
Please provide more description in the commit message as to the intent of these changes.  "Add support of multiple recv buffer repost" isn't clear given the changes being made.